xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-webjar/src/main/vue/layouts/table/LayoutTableHeaderNames.vue

```diff
@@ -181,6 +185,34 @@ export default {
         if (th.style.display !== "none") return th;
       }
     },
+    
+    keyboardDragNDropRight (e) {
+      let handles = e.currentTarget.closest('tr').querySelectorAll('.handle');
+      let oldIndex = Array.from(handles).indexOf(e.currentTarget);
+      let newIndex = oldIndex + 1;
+      if (newIndex < handles.length) {
+          this.logic.reorderProperty(oldIndex, newIndex);
+      } else {
+          this.logic.reorderProperty(oldIndex, 0);
```

Comment:
I don't think having content wrap around is unexpected behavior. When I am selecting the rightmost column and ask to move it to the right, I expect to have a shortcut to put it back to the leftmost position. This is just a convenience that can benefit power users: when I have 20 columns and I want to put the 13th in position 1, I could just go right 8 times and loop around, instead of going left 12 times. 
If I want to put the 20th column in 1st place, instead of going 20 times to the left, I can move it once to the right. Personally I think this is great QoL for this feature.

It's not the main point of the PR so I don't mind reverting it if we don't agree :)

Reply:
To me, it matches the mouse actions, you also cannot move the column further to the right with the mouse. But I don't mind that much.